body/stretch_body/stepper.py

```diff
@@ -680,8 +632,11 @@ def pack_trigger(self, s, sidx):
             sidx += 4
             return sidx
 
+    def unpack_status(self,s):
+        pass
```

Comment:
Would be better to `raise NotImplementedError()` here.

Reply:
Fixed